src/main/java/org/dependencytrack/model/Project.java

```diff
@@ -325,6 +326,8 @@ public enum FetchGroup {
 
     private transient String author;
 
+    private transient boolean active;
```

Comment:
I think this field is no longer needed, since `isActive` and `setActive` operate on the `inactiveSince` field.